### DISCUSSION TITLE
Strange behavior in daemon mode using dbus interface

### DISCUSSION THREAD

**Discussion by R0Y4LTS** [AsamK/signal-cli#1467] [Q&A]
After recognizing that my smart home  central (Raspi 3) wasn't able to send signal messages anymore, I had to change to v11.11 from  v11.06 due to a new signal server api version.
After the update it still couldn't send messages. I use this Python script based on the example script.

```python
# Bibliotheken laden
from pydbus import SystemBus
from gi.repository import GLib
import sys

bus = SystemBus()
loop = GLib.MainLoop()
signal = bus.get('org.asamk.Signal', object_path='/org/asamk/Signal')

def sendMsgToReceiversGroup(pMsg, pReceivers):
    for receiver in pReceivers:
        signal.sendMessage(pMsg, [], receiver)
        
if len(sys.argv) > 2:
    sendMsgToReceiversGroup(sys.argv[1], sys.argv[2].split(","))
``` 

So calling the script with

`python Send_signal_msg.py "Message", +XXX`

didn't work.

Surprisingly 

`signal-cli --dbus-system send -m "Message" +XXX`

worked.

After starting the daemon in verbose mode to debug that behavior both methods worked.
Any ideas? Has anybody observed this?
Looks like a timing issue to me.

**Comment by greenpines65** [comment]
Hi @R0Y4LTS,
The problem is described here: https://github.com/AsamK/signal-cli/discussions/1302#discussioncomment-9104254

Basically, what you see is actually several attempts to send your message to invalid numbers.
Please be careful and check that you haven't sent any sensitive information to unwanted recipients!
